date/index.js

```diff
@@ -5,14 +5,14 @@ import 'moment-timezone/moment-timezone-utils';
 export const settings = window._wpDateSettings;
 
 // Create WP timezone based off dateSettings.
-const offsets = ( settings.timezone.string ) ? momentLib.tz.zone( settings.timezone.string ).offsets : [ -settings.timezone.offset * 60 ];
-const momentTimezone = {
-	name: 'WP',
-	abbrs: [ 'WP' ],
-	untils: [ null ],
-	offsets: [ offsets ],
-};
-momentLib.tz.add( momentLib.tz.pack( momentTimezone ) );
+if ( settings.timezone.offset ) {
```

Comment:
I can't test now but I recall the special test was here for special timezones (like UTC)

Reply:
Hi @youknowriad, you are totally right this code had a bug.
The offset can be 0 in which the condition would return false and the code would not be executed. I removed the condition as if we don't initialize the timezone we have errors. I added an or to use 0 if we are in the presence of some undefined case (should never happen because  get_option( 'gmt_offset', 0 ); should always return something.